src/main/java/org/kainos/ea/db/ResponsibilityDao.java

```diff
@@ -16,29 +16,26 @@ public class ResponsibilityDao {
     private DatabaseConnector databaseConnector = new DatabaseConnector();
 
     public List<Responsibility> getRoleResponsibilityById(int id) throws SQLException {
```

Comment:
Why does getResposibilityById return a list? Should it not return a single object that matched the id provided?

Reply:
getResponsibilityById is intended to get a list of responsibilities from the database based on what roleId is assigned to it, although atm our test data doesnt have multiple for a single job role (ill add more sample data to the DB in a bit)

Reply:
Ahhh ok, makes sense.